services/app-web/src/pages/StateSubmission/StateSubmissionForm.tsx

```diff
@@ -127,63 +197,81 @@ export const StateSubmissionForm = (): React.ReactElement => {
         return specificContent ?? <GenericError />
     }
 
-    if (draft === undefined || draft === null) {
+    if (formDataError === 'MALFORMATTED_DATA') {
+        return <GenericError />
+    }
+
+    if (formDataError === 'NOT_FOUND' || (fetchData &&  !formDataFromLatestRevision)) {
         return <Error404 />
     }
 
+    if (formDataError === 'WRONG_SUBMISSION_STATUS') {
+        return <ErrorInvalidSubmissionStatus />
+    }
+
+    const draft = formDataFromLatestRevision as DraftSubmission
+
+    // const PageBannerAlerts = (): JSX.Element => {
+    //     const message = (typeof showPageErrorMessage !== 'boolean' ? showPageErrorMessage : undefined)
+    //     return (
+    //         <>
+    //             {showPageErrorMessage && (
+    //                 <FormAlert message={message}/>
+    //             )}
+    //             {unlockedInfo && (
+    //                 <SubmissionUnlockedBanner
+    //                     userType={
+    //                         loggedInUser?.role === 'CMS_USER'
+    //                             ? 'CMS_USER'
+    //                             : 'STATE_USER'
+    //                     }
+    //                     unlockedBy={unlockedInfo?.updatedBy || 'Not available'}
+    //                     unlockedOn={unlockedInfo.updatedAt || 'Not available'}
+    //                     reason={unlockedInfo.updatedReason || 'Not available'}
+    //                 />
+    //             )}
+    //         </>
+    //     )
+    // }
     return (
         <>
             <div className={styles.stepIndicator}>
                 <DynamicStepIndicator
                     formPages={activeFormPages(draft)}
                     currentFormPage={currentRoute}
                 />
+                {/* <PageBannerAlerts /> */}
             </div>
             <StateSubmissionContainer>
                 <Switch>
                     <Route path={RoutesRecord.SUBMISSIONS_TYPE}>
                         <SubmissionType
                             draftSubmission={draft}
                             updateDraft={updateDraft}
-                            formAlert={
```

Comment:
Why did we used to pass formAlert down?

Reply:
It used to be displayed inside the form container. I've moved all alerts outside now since the info alert designs show it outside makes sense  (to me) to keep alerts at the same level so they are easier to grok. Now only error summary errors will be in the form. what do you think?

Reply:
^ I looked more into this and I'm gonna push a commit just making a new shared generic alert banner  in `/components`. Thats what we display when the api fails with 500, we show this red alert. Realized we could be sharing that between pages including outside the state submission form. I will post a photo of what it looks like when you have a unlocked submission and you have api error both displayed and tag design when the work is up on this PR

Reply:
sounds great

Reply:
![Screen Shot 2022-03-14 at 5 28 38 PM](https://user-images.githubusercontent.com/10750442/158271471-8f383c8b-0359-4b7e-9ec7-50bcfeb496b7.png)

Here we go. This is what it would look like if an unlocked submission failed to update on "Continue" click for example. The  alerts will stack, with the error on top. Right now using the generic error message for almost everything in state submission form that can go wrong, but we can add more custom error messaging as we need it. cc: @joshmfranklin @MegMurphy